examples/README.md

```diff
@@ -0,0 +1,88 @@
+# Finch examples
+
+Examples included in Finch are scan path creation and various versions of a single line additive case.
+
+
+# Finch inputs
+
+All inputs are passed to Finch in a JSON format.
+
+## Temporal parameters (`time`)
+
+- `start_time`: Simulation start time
+  - units: `s`
+- `end_time`:	Simulation end time
+  - units: `s`
+- `Co`:	Courant number
+  - units: unitless
+- `total_output_steps`: Frequency to output grid information
+  - units: unitless
+- `total_monitor_steps`: Frequency to output timing information
+  - units: unitless
+
+## Spatial parameters (`space`)
+
+- `initial_temperature`: Initial temperature at all grid points
+  - units: `K`
+- `cell_size`: Grid cell size (per dimension)
```

Comment:
(per dimension) seems confusing here, a user may think this means that we support cell sizes that are non-cubic